packages/typography/src/Link/shared.styles.ts

```diff
@@ -21,6 +21,10 @@ export const linkStyles = css`
   cursor: pointer;
   font-size: inherit;
   line-height: inherit;
+  appearance: none;
+  background: none;
+  border: none;
+  padding: 0;
```

Comment:
`all: unset` is a cool shorthand for resetting https://developer.mozilla.org/en-US/docs/Web/CSS/all